src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/EditIssuePage.kt

```diff
@@ -36,9 +38,20 @@ class EditIssuePage(
                 .overwriteIfPresent("description")
         }
         form.fillRequiredFields()
+        selectResolution()
```

Comment:
We gotta let `fillRequiredFields` select the resolution for us, if it's required.